<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userWindow</class>
 <widget class="QWidget" name="userWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>个人中心</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ico_src.qrc">
    <normaloff>:/mainWindow_ico/user_ico.png</normaloff>:/mainWindow_ico/user_ico.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(195, 227, 218)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_8" stretch="1,4,4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
       <item>
        <widget class="QPushButton" name="user_back_but">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>幼圆</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(125, 200, 203);
border-radius: 3px;</string>
         </property>
         <property name="text">
          <string>返回</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/mainWindow_ico/icon/back_icon_w.png</normaloff>:/mainWindow_ico/icon/back_icon_w.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="topMargin">
        <number>50</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>400</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0">
           <item>
            <widget class="QLabel" name="user_headportrait_icon">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>132</width>
               <height>132</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-radius:5px;
background-color:rgb(170, 170, 255)</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="user_name_label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>129</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>昵称</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>400</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="user_coupon_number">
             <property name="font">
              <font>
               <family>幼圆</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(168, 221, 217)</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="user_coupon_name">
             <property name="font">
              <font>
               <family>幼圆</family>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(168, 221, 217)</string>
             </property>
             <property name="text">
              <string>优惠券</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="user_account_number">
             <property name="font">
              <font>
               <family>幼圆</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(168, 221, 217)</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="user_account_name">
             <property name="font">
              <font>
               <family>幼圆</family>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(168, 221, 217)</string>
             </property>
             <property name="text">
              <string>账户</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="user_integration_number">
             <property name="font">
              <font>
               <family>幼圆</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(168, 221, 217)</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="user_integration_name">
             <property name="font">
              <font>
               <family>幼圆</family>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(168, 221, 217)</string>
             </property>
             <property name="text">
              <string>积分</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="user_reward_name">
             <property name="font">
              <font>
               <family>幼圆</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(168, 221, 217)</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="user_reward_number">
             <property name="font">
              <font>
               <family>幼圆</family>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(168, 221, 217)</string>
             </property>
             <property name="text">
              <string>奖励金</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="user_moreinf_label">
         <property name="font">
          <font>
           <family>幼圆</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>更多信息请前往小程序查询...</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ico_src.qrc"/>
 </resources>
 <connections/>
</ui>
